spec/services/github_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe Github do
+  context "when given a repo" do
+    it 'find pull requests' do
+      VCR.use_cassette('empirical_oct_prs') do
+        prs = Github.pull_requests_by_repo('empirical-org', 'empirical-core', '2014-10-01', '2014-10-02')
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.